test/check.js

```diff
@@ -7,5 +7,5 @@ var postcss = require('postcss');
 module.exports = function check( input, output, opts ) {
   var processor = postcss([lost(opts)]);
 
-  expect(processor.process(input).css).to.equal(output);
```

Comment:
@wbruno Any thoughts to this syntax is correct but the output is incorrect? ...am I mistaken in my understanding of what the output is suppose to be?